app/routers/profile.py

```diff
@@ -60,6 +62,36 @@ async def profile(
     })
 
 
+@router.get("/start_exercise")
+async def start_exercise(session: Session = Depends(get_db)):
+    user = session.query(User).filter_by(id=1).first()
+
+    # Update database
+    user.is_active_exercise = True
+    session.commit()
+
+    # create user exercise
+    create_user_exercise(session, user)
+    url = router.url_path_for("profile")
+    return RedirectResponse(url=url, status_code=HTTP_302_FOUND)
+
+
+@router.get("/stop_exercise")
+async def stop_exercise(session=Depends(get_db)):
+    """
```

Comment:
The docstrings are for the users, you shouldn't explain here what the code does.

Reply:
Ok Remove the comment